pytorch_lightning/trainer/trainer.py

```diff
@@ -1699,6 +1699,11 @@ def world_size(self) -> int:
         # some training types define a world size
         return getattr(self.training_type_plugin, "world_size", 1)
 
+    @property
+    def should_rank_save_checkpoint(self) -> bool:
+        rank_zero_deprecation("`Trainer.should_rank_save_checkpoint` is deprecated in v1.6 and will be removed in 1.8.")
```

Comment:
```suggestion
        rank_zero_deprecation("`Trainer.should_rank_save_checkpoint` is deprecated in v1.6 and will be removed in 1.8.", stacklevel=5)
```

Properties should increase the stacklevel by 1.